code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm

```diff
@@ -1444,6 +1444,7 @@
 	display_name = "Lucky's amour"
 	ckeywhitelist = list ("thedavestdave")
 	character_name = list("Lucky")
+	allowed_roles = list("Chaplain")
```

Comment:
This should hopefully fix it, and ensure it stays as a chaplain only thing @Very-Soft